script/Ciel/Day 10/10_01_CIEL10_3.txt

```diff
@@ -53,17 +60,21 @@ But still... I said it.
 -- 言いたい事、聞きたい事は山ほどあるけど、この気持ちを伝えられたのなら、今はそれで充分だ。
 // What's the worst that could happen? She rejects me, right?
 // Her: Ewwww
-There's a mountain of things I would have like to say and to ask, but I will be satisfied if my feelings have been correctly conveyed.
+I still have all sorts of things I want to tell her and ask her about,
+ but if I've managed to convey the way I feel,
+ that's all I could hope for right now.
 }
 [sha:23b385335c5d17770a90bd17b9abab7a70a7b5a3]{
 -- Page 131, Offset 27415.
 -- 「ありがとうございます。今のお返事は、素直に、びっくりするぐらい嬉しいです。
-"Thank you. This answer you've given me is honestly making me so happy I can barely believe it myself.
+"Thank you.
+ I mean it when I say that you've made me so happy I can hardly believe it.
```

Comment:
Hmm, the latter sentence almost sounds kinda insincere the way its currently worded.

>When you put it like that... it really does make me almost unbelievable happy.

Well, something like that.